README.md

```diff
@@ -125,7 +125,7 @@ Gollum comes with the following command line options:
 | --no-display-metadata | none  | Do not render metadata tables in pages. |
 | --user-icons      | [MODE]    | Tell Gollum to use specific user icons for history view. Can be set to `gravatar`, `identicon` or `none`. Default: `none`. |
 | --mathjax-config  | [FILE]    | Specify path to a custom MathJax configuration. If not specified, uses the `mathjax.config.js` file from repository root. |
-| --template-dir    | [PATH]    | Specify custom mustache template directory. |
+| --template-dir    | [PATH]    | Specify custom mustache template directory. Only templates you wish to override need to exist in this path. |
```

Comment:
I don't think including "you" is too bad.

If you are still unhappy with it, I might suggest:

```
Only overrides templates that exist in this directory.
```